.framework/node/backend/package.json

```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "start": "node ./app.js",
+    "start-test": "export WILCO_ID=0 && export MONGODB_URI=mongodb://localhost/anythink-market && node ./app.js ",
```

Comment:
maybe anythink-market-test ? And then your tests will first clear the DB, fill it with dummy data and start tests.

Reply:
yep, can do. I actullay prefer to hide the fact that we override wilcoId, better if users want play with it. I'll check how I can hide it using the github action itself